draft-ietf-rats-eat.md

```diff
@@ -1198,103 +1150,42 @@ The string identifying the JSON-encoded token MUST be one of the following:
 : The second array item must be some base64url-encoded CBOR that is a tag, typically a CWT or CBOR-encoded detached EAT bundle
 
 "BUNDLE":
-: The second array item MUST be a JSON-encoded detached EAT bundle as defined in this document.
-
-Additional types may be defined by a standards action.
-
-When decoding, the array of two is decoded.
-The first item indicates the type and encoding of the nested token.
-If the type string is not "CBOR", then the token is JSON-encoded and of the type indicated by the string.
-
-If the type string is "CBOR", then the token is CBOR-encoded.
-The base64url encoding is removed.
-The CBOR-encoded data is then decoded.
-The type of nested token is determined by the CBOR-tag.
-It is an error if the CBOR is not a tag.
-
-~~~~CDDL
-{::include nc-cddl/nested-token-json.cddl}
-~~~~
-
-
-##### Detached Submodule Digest
-
-This is type of submodule equivalent to a Claims-Set submodule, except the Claims-Set is conveyed separately outside of the token.
+: The second array item MUST be a JSON-encoded Detached EAT Bundle as defined in this document.
 
-This type of submodule consists of a digest made using a cryptographic hash of a Claims-Set.
-The Claims-Set is not included in the token.
-It is conveyed to the verifier outside of the token.
-The submodule containing the digest is called a detached digest.
-The separately conveyed Claims-Set is called a detached claims set.
+Both the submodule Claims-Set and detached Claims-Set options use the same attesting environment as the surrounding token.
```

Comment:
Maybe remove this and let this section just be about encoding / decoding. Let the following sections handle this.

Reply:
Removed.